packages/sdk/src/constants/CicsCmci.constants.ts

```diff
@@ -48,6 +58,11 @@ export const CicsCmciConstants = {
      */
   CICS_DEFINITION_WEBSERVICE: "CICSDefinitionWebService",
 
+  /**
+     * Specifies the required part of the REST interface URI to access tcp/ip service definitions
+     */
+  CICS_DEFINITION_TCPIPSERVICE: "CICSTCPIPService",
+
```

Comment:
This is not accurate. The comment and constant name suggests definitions, but the actual resource name suggests the installed resource. Which do you need here?